File: Flask/microservices/place-service/website/dto/get_places_response.py
from typing import List
from website.place import Place


def entity_to_dto_mapper(places: List[Place]):
    return GetPlacesResponse(places).places


class GetPlacesResponse:
    def __init__(self, places: List[Place]) -> None:
        self.places = [place.as_dict() for place in places]

File: Spring Boot/microservices/place-service/src/main/java/pl/edu/pg/benchmarking/place/dto/GetPlacesResponse.java
package pl.edu.pg.benchmarking.place.dto;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Getter;

import java.util.Collection;
import java.util.function.Function;
import java.util.stream.Collectors;

public class GetPlacesResponse {

    @Getter
    @AllArgsConstructor(access = AccessLevel.PRIVATE)
    public static class Place {
        private Integer id;
        private String coordinates;
    }

    public static Function<Collection<pl.edu.pg.benchmarking.place.Place>, Iterable<Place>> entityToDtoMapper() {
        return places -> places.stream()
                    .map(place -> new Place(place.getId(), place.getCoordinates()))
                    .collect(Collectors.toList());
    }

}